src/js/view/controls/components/menu/menu.js

```diff
@@ -447,7 +450,10 @@ export default class Menu extends Events {
             }
         });
     }
-    toggle(evt) {
+    toggle(evt, isDefault) {
```

Comment:
can i get an overview on what this `isDefault` is doing?

Reply:
It's the param for if it's opened via the default action (clicking the settings icon). Passing this through fixes a toggle issue